src/development/coding-frontend.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: article
+title: "Frontends"
+tags: dev
+order: 11
+---
+## Accessibility
+
+Producing accessibile web sites and applications is a fundamental aspect of frontend development.
+
+The Government service manual explains more about [making your service accessible][making_your_service_accessible].
+
+The NHS service manual also provides guideance on [accessibility][nhs_accessibility] including specific advice on [design][nhs_accessibility_design] and [development][nhs_accessibility_development].
+
+As a frontend developer at the NHSBSA you should:
+
+* Use semantic HTML
+* Use aria roles for interactive elements
+* Build user interfaces that work by default without rich client technology such as Javascript
+* Build user interfaces that adopt progressive enhancement and not use graceful degradation
+  Government advice is to [design the user interfaces with progressive enhancement][gov_progressive_enhancement]
+
+## NHS and Gov design systems
+
+We make use of the [NHS design system][nhs_design_system] and associated [NHS frontend][nhs_frontend] package to align to standard styles, components and patterns that have been proven with robust user testing. In cases where Government branding is requird, we adopt the [Government design system][gov_design_system] and [frontend][gov_frontend].
+
+Both NHS and Gov frontends are implemented in Node.js and Nunjucks. Our preference should be to implement frontends in Node.js to take full advantage of the frontend packages.
+
+We occasionally need to adopt non-standard patterns when user research proves specific difficulties with the standard patterns. Any new pattern should be user tested and then fed back into the NHS or Gov design system and frontend package.
+
+## Welsh
```

Comment:
Should this section actually emphasise the importance of making the service multi-lingual, even if initially it's just English and Welsh.

A service that can only supports two languages hasn't be properly implemented.  Adding a third language in future should be as simple as adding a new toggle button and language files.